test/e2e/autoscaling/horizontal_pod_autoscaling.go

```diff
@@ -67,7 +67,16 @@ var _ = SIGDescribe("[HPA] Horizontal pod autoscaling (scale resource: CPU)", fu
 	})
 
 	SIGDescribe("ReplicationController light", func() {
-		ginkgo.It("Should scale from 1 pod to 2 pods", func() {
+		/*
+		  Release : v1.16
+		  Testname: Horizontal Pod Autoscaling, CPU Target Utilization Scale
+		  Description: Given 1 Pod with 150mCPU usage
+		  And 200mCPU per Pod request
+		  And targeted CPU uitilzation is 50%
+		  Then the HP Autoscaler will create another pod
+		  And the 150mCPU usage will be spread across both Pods
+		*/
+		framework.ConformanceIt("Should scale from 1 pod to 2 pods", func() {
```

Comment:
Please lowercase Should so it is like all the others.

Also the description now is accurate but it's still not in the right format - [RFC2119](https://tools.ietf.org/html/rfc2119). So I think this would be better:

Given 1 Pod with 150mCPU usage, 200mCPU per Pod request, and targeted CPU utilization of 50%, the HP Autoscaler MUST create another pod, spreading the 150mCPU usage across both Pods.